apps/web/src/pages/EmploymentEquityPage/dialogs/IndigenousDialog.tsx

```diff
@@ -9,30 +14,120 @@ import {
   getEmploymentEquityGroup,
   getEmploymentEquityStatement,
 } from "@common/constants";
-import type { EquityDialogProps } from "../types";
+import { getSelfDeclarationLabels } from "~/components/SelfDeclaration/utils";
+import { CommunityList } from "~/components/SelfDeclaration/CommunitySelection";
+import { Fieldset } from "~/../../../frontend/common/src/components/inputPartials";
+import { FieldLabels } from "~/../../../frontend/common/src/components/form/BasicForm";
```

Comment:
Kind of annoying but auto imports for some reason use the `~` instead of `@common` so this could be changed to (you may want to confirm though):

```suggestion
import { Fieldset } from "@common/components/inputPartials";
import { FieldLabels } from "@common/components/form/BasicForm";
```

Reply:
Nice! I think if we do #5072 this should resolve itself 👍 For now, typescript loves using the `~` for auto imports in vs code 😢